build.go

```diff
@@ -68,6 +71,139 @@ func if_not_exists(cmd string, cb func()) {
 	}
 }
 
+func removeExt(s string) string {
+	return s[:len(s)-len(filepath.Ext(s))]
+}
+
+func unzip(src, dest, fileToExtract string) error {
+	r, err := zip.OpenReader(src)
+    if err != nil {
+        return err
+    }
+    defer func() {
+        if err := r.Close(); err != nil {
+            panic(err)
+        }
+    }()
+
+    os.MkdirAll(dest, 0755)
+
+    // Closure to address file descriptors issue with all the deferred .Close() methods
+    extractAndWriteFile := func(f *zip.File) error {
+       rc, err := f.Open()
+        if err != nil {
+            return err
+        }
+        defer func() {
+            if err := rc.Close(); err != nil {
+                panic(err)
```

Comment:
ok, right, I remember now. This is a deferred file close with potentially more than one. Panic lets all deferred operations finish before exiting and deferred functions execute right before close. So this should be a safe panic and has a reason.